src/storage/rocksdb/src/main/java/org/locationtech/geogig/rocksdb/RocksdbObjectStore.java

```diff
@@ -205,7 +205,7 @@ public boolean exists(ObjectId id) {
 
     private boolean exists(RocksDBReference dbRef, ReadOptions readOptions, byte[] key) {
         int size = RocksDB.NOT_FOUND;
-        if (dbRef.db().keyMayExist(key, new StringBuffer())) {
+        if (dbRef.db().keyMayExist(key, new StringBuilder(0))) {
```

Comment:
this might want to be key.length (instead of 0). Not sure what it does - but might be slightly faster to not have to re-allocate the memory.  Not worth worrying about - just noticed it.